common/anomalies/hfe_anomaly.txt

```diff
@@ -62,24 +67,26 @@ hfe_anomaly_criopods = {
 			}
 		}
 		modifier = {
-			factor = 2
+			factor = 1.5
+			hfe_trigger_story_enabled = yes
```

Comment:
You dont need this trigger

Reply:
You specifically told me to add that trigger, remember?

Reply:
No, i told you how add work, with multiplication it is not necessary

Reply:
Okay, done